packages/app/src/commands/i18n/modernize.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @example
+ *
+ * npx -p @dhis2/cli d2-app i18n modernize \
+ *   ~/development/dhis2/maintenance-app/src/i18n # inDir \
+ *   ~/development/dhis2/project-doom/i18n # outDir \
+ *   --override-existing-files \
+ *   --pootle-path "/app/import-export-app/" \
+ *   --languages fr,ar \
+ *   --delete-old-files \
+ *   --log-missing-keys
+ */
```

Comment:
_(minor)_ this could just be `npx @dhis2/cli app i18n modernize` or `npx @dhis2/cli-app i18n modernize`, there shouldn't be a need for the `-p` or `d2-app`